bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -38,10 +38,10 @@
     /**
      * Exploded jenkins.war
      */
-    @Option(name = "-w", aliases = { "--jenkins-war" }, usage = "path to jenkins.war or exploded jenkins war directory", forbids = { "-v" })
+    @Option(name = "-w", aliases = { "--jenkins-war" }, usage = "path to exploded jenkins war directory", forbids = { "-v" })
     public File warDir;
 
-    @Option(name = "-v", aliases = { "--version"}, usage = "jenkins version to use. Defaults to latest LTS.")
+    @Option(name = "-v", aliases = { "--version"}, usage = "jenkins version to use (only in case 'warDir' is null). Defaults to latest LTS.")
```

Comment:
"Is not specified". Null does not help in user context w/o loojing at the code

Reply:
Changed.